src/parameterizations/lateral/MOM_tidal_forcing.F90

```diff
@@ -527,8 +534,46 @@ subroutine tidal_forcing_init(Time, G, US, param_file, CS, HA_CS)
     enddo
   endif
 
+  call get_param(param_file, mdl, "TIDE_ADD_NODAL", add_nodal_terms, &
+                 "If true, include 18.6 year nodal modulation in the astronomical tidal forcing.", &
+                 default=.false.)
+  call get_param(param_file, mdl, "TIDE_NODAL_REF_DATE", nodal_ref_date, &
```

Comment:
Is there a reasonable scenario that nodal correction should use a date other than `TIDE_REF_DATE`?
I appreciate the flexibility for introducing a different reference date for nodal correction, but it would also seem rather confusing.

Reply:
This is what has been done in `MOM_open_boundary.F90`. I think the reason is that these two dates could be different when tides are forced at the open boundaries by some external data. @andrew-c-ross probably has a better answer.

Reply:
When I wrote the original OBC tide code I wasn't aware of an easy way for MOM6 to know the current date/time when the OBC and tidal forcing were being initialized. If `Time_init` is available though I think it would make sense to use that for both the tide and nodal correction ref dates. The only exception would be if, when restarting the model, the init time was something other than the time of the restart, since we would want to update the date used for the nodal correction ~yearly.

Reply:
Thanks for the answers!